tgui/package.json

```diff
@@ -39,7 +39,7 @@
     "postcss-filter-gradient": "0.2.2",
     "postcss-font-weights": "2.0.1",
     "postcss-opacity": "3.0.0",
-    "ractive": "0.7.3",
+    "ractive": "mrstonedone/ractive-build#8.0",
```

Comment:
This format converts to a github url, so thats pulling the tag 8.0 from the ractive-build repo on my github.

This could be moved to a tgstation org build if needed.